src/source/PeerConnection/PeerConnection.c

```diff
@@ -921,6 +921,21 @@ STATUS peerConnectionGetCurrentLocalDescription(PRtcPeerConnection pRtcPeerConne
     return retStatus;
 }
 
+/// @brief parses string of form "$number $whatever" returns $number as uint32
+/// @return 0 if value is not parsable or null
+static UINT32 parseExtId(PCHAR extmapValue)
+{
+    if (extmapValue == NULL)
+        return 0;
+    PCHAR split = STRCHR(extmapValue, ' ');
+    if (split == NULL)
+        return 0;
+    UINT32 extid = 0;
```

Comment:
extid/split also need to be declared up top, it looks like the CI doesn't check for this.  The established pattern (and I apologize because it might not be documented) is conditional statements ALWAYS have {} even if it's single line.  Otherwise looks great!